src/Documenter.jl

```diff
@@ -389,8 +392,6 @@ function deploydocs(;
                         """
                         echo "$(Compat.String(base64decode(documenter_key)))" >> $keyfile
```

Comment:
Just a thought that crossed my mind -- would it perhaps be better to directly write the key to a file, instead of interpolating it into a script first? I [did that while debugging](https://github.com/JuliaDocs/Documenter.jl/blob/mp/deploy-hang-debug/src/Documenter.jl#L382) since I wanted to see the contents of the script in the log for debugging.